modules/tinymce/src/core/test/ts/browser/EditorTest.ts

```diff
@@ -45,6 +45,24 @@ describe('browser.tinymce.core.EditorTest', () => {
     editor.off('change');
   });
 
+  it('TINY-6920: Do not fire change event at first typed character', () => {
+    const editor = hook.editor();
+    let changeEventCounter = 0;
+
+    const onChange = () => {
+      changeEventCounter++;
+    };
+
+    editor.setContent('');
+    editor.setDirty(false);
```

Comment:
```suggestion
    editor.resetContent('');
```
This is a better way to reset the content, dirty state and clear the undomanager state.

Reply:
Useful.